pkg/proxy/proxy.go

```diff
@@ -45,28 +63,27 @@ func getLoadTestType(r *http.Request) apisLoadTestV1.LoadTestType {
 	return apisLoadTestV1.LoadTestType(r.FormValue(backendType))
 }
 
-//CreateLoadTestHandler creates loadtest CR on POST request
-func CreateLoadTestHandler(loadTestClient loadTestV1.LoadTestInterface, maxLoadTestsRun int) http.Handler {
+//Create creates loadtest CR on POST request
+func (p *Proxy) Create() http.Handler {
```

Comment:
All proxy methods should expect writer and request as input like this:
```suggestion
func (p *Proxy) Create(w http.ResponseWriter, r *http.Request) {
```